.eslintignore

```diff
@@ -15,4 +15,4 @@ src/lib/backend.ai-client-node.ts
 src/lib/backend.ai-client-node-to-es6.js
 src/lib/backend.ai-client-node-test-native-es6.js
 src/lib/Chart.min.js
-src/lib/json_to_csv.js
+src/lib/json_to_csv.js
```

Comment:
Is there any reason to delete? If not, please restore.

Reply:
@yomybaby deleted code and new code is same. But I don't know why file is shown that was changed